sync_io/sync_io.mbti

```diff
@@ -0,0 +1,14 @@
+package moonbitlang/core/sync_io
+
+// Values
+fn read_file_to_bytes(path~ : String) -> Bytes!
+
+fn write_bytes_to_file(path~ : String, content~ : Bytes) -> Unit
+
+// Types and methods
+type IOError
+
```

Comment:
@fs_sync.read_file_to_bytes(path:String) -> Bytes!
@fs_sync.write_bytes_to_file(path:String, content:Bytes) -> Unit! // may error when file not exists etc
@fs_sync.read_file(path: String, encoding : String = "utf8") -> String!
@fs_sync.write_string_to_file(path:String, encoding: String="utf8") -> Unit!